task8-main/src/main/java/com/example/classloader/app/Demo.java

```diff
@@ -0,0 +1,82 @@
+package com.example.classloader.app;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.*;
+
+public class Demo {
+    private static final Logger LOGGER = LoggerFactory.getLogger(Demo.class);
+
+    public static void main(String[] args) throws IOException, ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException {
+
+        String pathName = getModulsLocation("location.properties");
+        List<File> jars = Arrays.asList(new File(pathName).listFiles());
+        URL[] urls = new URL[jars.size()];
+        for (int i = 0; i < jars.size(); i++) {
+            try {
+                urls[i] = jars.get(i).toURI().toURL();
+                System.out.println(urls[i]);
+            } catch (Exception e) {
```

Comment:
never catch Exception. Use more precise inhereted classes. Try to handle exception more elegant